spec/classroom_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../classroom'
+
+describe Classroom do
+  describe '#initialize' do
+    it 'Creates a new classroom' do
+      classroom = Classroom.new('Physics')
+      expect(classroom).to be_a(Classroom)
+    end
+  end
+
+  describe 'check attributes' do
+    it 'checks name' do
+      classroom = Classroom.new('Physics')
+      expect(classroom.label).to eq('Physics')
+    end
+  end
+
+  describe 'check methods' do
```

Comment:
- [ ] Kindly rewrite this test is failing.
<img width="1016" alt="Screenshot 2023-06-10 at 01 34 57" src="https://github.com/evillaz/school-library/assets/81444338/98d3d086-934b-466d-bc88-fb4eed4855b0">